src/battle-log.ts

```diff
@@ -141,9 +141,9 @@ class BattleLog {
 
 		case 'unlink': {
 			const user = toID(args[2]) || toID(args[1]);
-			this.unlinkChatFrom(user);
+			this.unlinkChatFrom(user, parseInt(args[args.length], 10));
 			if (args[2]) {
-				this.hideChatFrom(user);
+				this.hideChatFrom(user, parseInt(args[args.length], 10));
```

Comment:
```suggestion
				const lineCount = parseInt(args[args.length], 10);
				this.hideChatFrom(user, true, lineCount);
```

Reply:
(You can't only use the third argument, you have to use the second and then the third.)

Reply:
The line count is always `args[3]`, because this is `|unlink|hide|USERNAME|LINECOUNT`.